analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.CSharp10.cs

```diff
@@ -5,14 +5,16 @@ namespace Tests.Diagnostics
 {
     public record struct RecordStruct
     {
+        public RecordStruct() { }
+
         private int a = 42; // Noncompliant {{Remove the unused private field 'a'.}}
 
         private int b = 42;
         public int B() => b;
 
         private nint Value { get; init; } = 42;
 
-        private nint UnusedValue { get; init; } // Noncompliant
+        private nint UnusedValue { get; init; } = 0; // Compliant - properties with initializer are considered as used.
```

Comment:
Possible duplicate of line 15.
But. What happens, if UnusedValue is set in the ctor instead?

Reply:
The behavior does not changes, the property value is considered used. Here the behavior of the rule does not changes, I did the change only to be able to compile the code.